pkg/plugins/client_test.go

```diff
@@ -31,24 +31,30 @@ func teardownRemotePluginServer() {
 	}
 }
 
-func TestHttpTimeout(t *testing.T) {
+func testHttpTimeout(t *testing.T, timeout, epsilon time.Duration) {
```

Comment:
Unrelated to the race fix, but I think this should be called `testHTTPTimeout`. For example, the `net/http` package has methods called `ServeHTTP`, not `ServeHttp`.